segway/gmtk/params.py

```diff
@@ -399,27 +526,71 @@ def __str__(self) -> str:
         return "\n".join(line)
 
     def get_header_info(self) -> str:
+        # No additional header information needed in input.master.
+        return ""
+
+
+class GenericString:
```

Comment:
Can't this be a `str` subclass to make life easier, just like `Array` is an `ndarray`?

Reply:
I've made this a string subclass so removed the explicit`__init__` and `__str__` methods.